### ISSUE TITLE
Donate prod default branch for heroku isn't master but the first branch alphabetically

### ISSUE THREAD

**Issue by ScottDowne** [MozillaFoundation/mofo-devops#542]
![huh](https://user-images.githubusercontent.com/197334/31150876-54f21086-a863-11e7-9ef9-7d5273517473.png)

Giddy also pointed out that it's not a dropdown like it should be.

Something changing on Heroku's end?

Either way having it be some rando branch is a pretty bad trap.

@cadecairos

**Comment by cadecairos** [comment]
Yeah it looks like they removed the drop down that it once was, and are auto-filling the input with the alphabetically first branch... I'm don't think there's much we can do, besides clean out the branches on the repo.

**Comment by gideonthomas** [comment]
@cadecairos I don't see this in other apps though (for e.g. `bramble-production`)

**Comment by cadecairos** [comment]
Oh, that's weird. I can't see any obvious difference between the two apps or repos.

I disconnected the pipeline from the repo and re-connected it, but the issue persisted. I can open a support ticket.

**Comment by cadecairos** [comment]
we discovered a bug in their dashboard!

From Heroku support:

```
Hi Christopher,

This is indeed a bug we'll be addressing shortly :) The reason you're seeing an input field rather than a dropdown is because this particular repo has 100+ branches on it -- our UI switches to input so that you don't have to scroll through such a long list.

Apologies for the inconvenience but thank you for taking the time to let us know!

Warm regards,
Johnny
```
